src/MonoTorrent/MonoTorrent.Client/ClientEngine.cs

```diff
@@ -77,6 +78,9 @@ public class ClientEngine : IDisposable
         public event EventHandler<StatsUpdateEventArgs> StatsUpdate;
         public event EventHandler<CriticalExceptionEventArgs> CriticalException;
 
+        public event EventHandler<TorrentEventArgs> TorrentRegistered;
+        public event EventHandler<TorrentEventArgs> TorrentUnregistered;
```

Comment:
This looks like a good addition. Could this, and related changes, be added in a separate PR?